### PR TITLE
Feature/Add all abstract extraction rules

### PR DESCRIPTION
**Pull request by xkopenreview**
this pr should add rules for domains for 95% coverage as described in
https://github.com/openreview/openreview-js/pull/29#issuecomment-1957191812

and add tests for the rules

### CODE REVIEW

File: packages/meta-extraction/src/abstractExtractionRules.js [hunk 1]
```diff
@@ -144,11 +151,6 @@ const gatherDublinCoreTags = async (page) => {
       return { name, type, value };
     })
   );
-//   const evidences = await Promise.all(
-//     ['DC.Creator', 'DC.Subject', 'DC.Identifier', 'DC.Type'].map(
-//       async (name) => selectAllMetaEvidence(page, name)
-//     )
-//   );
```
Comment (carlosmondra) [inline packages/meta-extraction/src/abstractExtractionRules.js:151]: Can we delete this commented code?

File: packages/meta-extraction/test/test.js [hunk 2]
```diff
@@ -32,15 +32,15 @@ describe.only('Abstract Extraction', function () {
     assert.equal(pdf,pdfExpected);
   });
 
-  it.skip('should extract abstract and pdf using aaai rule (aaai.org)', async function () {
+  it('should extract abstract and pdf using aaai rule (aaai.org)', async function () {
```
Comment (carlosmondra) [inline packages/meta-extraction/test/test.js:35]: Why are we skipping AAAI tests?

File: packages/meta-extraction/src/abstractExtractionRules.js (hunk 1 above)
Comment (xkopenreview) [reply packages/meta-extraction/src/abstractExtractionRules.js:151]: this commented code is deleted isn't it?

File: packages/meta-extraction/test/test.js (hunk 2 above)
Comment (xkopenreview) [reply packages/meta-extraction/test/test.js:35]: i think the change to from skip to not skip

Comment (carlosmondra) [reply packages/meta-extraction/test/test.js:35]: Oh, I completely missed that. Thanks!

File: packages/meta-extraction/src/abstractExtractionRules.js (hunk 1 above)
Comment (carlosmondra) [reply packages/meta-extraction/src/abstractExtractionRules.js:151]: OMG! I just wanted to write comments in your PR apparently.

**Pull request merged**